File: Server/think/application/spider/model/QidianBookModel.php
<?php
/**
 * Created by PhpStorm.
 * User: Arthur
 * Date: 2016/10/24
 * Time: 上午9:44
 */

namespace app\spider\model;


use app\spider\model\ModelWrap;

class QidianBookModel extends ModelWrap
{
    // 设置当前模型对应的完整数据表名称
    protected $table = 'book_qidian';

    protected $pk = 'id';

    public $id;
    public $URL;
    public $imgURL;
    public $shortDescription;
    public $longDescription;
    public $name;
    public $authorURL;
    public $authorName;
    public $mainCategory;
    public $subCategory;
    public $status;
    public $writtenWords;


}

File: iOS/BookReader/BookReader/Common/WebAPI/Model/QiDianBookModel.swift
//
//  QiDianBookModel.swift
//  BookReader
//
//  Created by Arthur on 2016/10/28.
//  Copyright © 2016年 Arthur. All rights reserved.
//

import UIKit
import ObjectMapper

class QiDianBookModel: Mappable {
    var id: Int?
    var URL: String?
    var imgURL:String?
    var shortDescription:String?
    var longDescription:String?
    var name:String?
    var authorURL:String?
    var authorName:String?
    var mainCategory:String?
    var subCategory:String?
    var status:String?
    var writtenWords:String?
    
    required init?(_ map: Map) {
        
    }
    
    func mapping(map: Map) {
        id <- map["id"]
        URL <- map["URL"]
        imgURL <- map["imgURL"]
        shortDescription <- map["shortDescription"]
        longDescription <- map["longDescription"]
        name <- map["name"]
        authorURL <- map["authorURL"]
        authorName <- map["authorName"]
        mainCategory <- map["mainCategory"]
        subCategory <- map["subCategory"]
        status <- map["status"]
        writtenWords <- map["writtenWords"]
    }
}
